internal/pkg/kv/db/sql/sqliteDatabase.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package sqlkv
+package sql
```

Comment:
how about sqlite, we might support other dbms in future?

Reply:
I refactored it